flight/PiOS/Common/pios_mpu.c

```diff
@@ -0,0 +1,1090 @@
+/**
+ ******************************************************************************
+ * @file       pios_mpu.c
```

Comment:
Yeah, great point. We just had Invensense reps stop by this morning and they have a ton of new and interesting sensors beyond the MPU line.